cnxarchive/__init__.py

```diff
@@ -33,114 +28,45 @@
     'content-type',
     ]
 
+# XXX (25-Jul-2015) Should be removed ASAP
 def get_settings():
```

Comment:
Let's add an issue for this. It's one of those things that shouldn't be left hanging around.